--- a/quickshell/Modules/DankBar/Widgets/ControlCenterButton.qml	2026-08-23 14:12:14.378473651 +0000
+++ b/quickshell/Modules/DankBar/Widgets/ControlCenterButton.qml	2026-08-23 14:16:00.074484413 +0000
@@ -631,7 +631,7 @@
                     visible: root.hasNoVisibleIcons()
 
                     DankIcon {
-                        name: "settings"
+                        name: "power_settings_new" // LIVARA_DMS_POWER_BUTTON
                         size: root.vIconSize
                         color: root.isActive ? Theme.primary : Theme.widgetIconColor
                         anchors.centerIn: parent
@@ -856,7 +856,7 @@
                 }
 
                 DankIcon {
-                    name: "settings"
+                    name: "power_settings_new" // LIVARA_DMS_POWER_BUTTON
                     size: root.getControlCenterIconSize()
                     color: root.isActive ? Theme.primary : Theme.widgetIconColor
                     anchors.verticalCenter: parent.verticalCenter
