diff --git a/plugin.toml b/plugin.toml
--- a/plugin.toml
+++ b/plugin.toml
@@ -28 +28 @@
-default = "portal"
+default = "focused"
diff --git a/recorder_service.luau b/recorder_service.luau
--- a/recorder_service.luau
+++ b/recorder_service.luau
@@ -274 +274 @@
-    local flags = `-w {source} -f {fps} -k {codec} {audioFlags} -bm qp -ffmpeg-video-opts qp={qp} -cursor {cursor} -cr {cr} {resFlag} {restore} -v no -o "{outputPath}"`
+    local flags = `-w {source} -f {fps} -k {codec} -fallback-cpu-encoding yes {audioFlags} -bm qp -ffmpeg-video-opts qp={qp} -cursor {cursor} -cr {cr} {resFlag} {restore} -v no -o "{outputPath}"`
